misc/format_material.py

```diff
@@ -20,6 +20,10 @@ def resize_imgs(input_dir):
         root = tree.getroot()
         root.attrib["width"] = "144"
         root.attrib["height"] = "144"
+        for child in root:
```

Comment:
I guess this is done to enforce transparency, but we should add a comment to explain this